style.css

```diff
@@ -330,112 +336,110 @@
 
 /* =================================================contact form========================================= */
 
-.contact-section{
+.contact-section {
   width: 375px;
-  height: 775px;
-  background:  #6070FF;;
-  border-top-left-radius:20%;
+  height: 779px;
+  background: #6070ff;
+  border-top-left-radius: 8%;
```

Comment:
- [ ] Great work on implementing the required changes suggested by the previous reviewer :+1:. However, as previously stated, the border-top left doesn't match the design in the Figma template. To address this, I recommend setting the `border-top-left-radius` to 100 pixels instead of 8 percent.

**Your design**

![Screenshot from 2023-04-05 07-45-53](https://user-images.githubusercontent.com/105117832/230085620-009dbccc-a5f8-441f-94e7-dd6290d926fe.png)

**Figma**

![Screenshot from 2023-04-05 07-52-00](https://user-images.githubusercontent.com/105117832/230085887-775fcf20-a065-4953-bc7f-3105a535ea4e.png)

Reply:
Thanks fixed the radius